dracut/anaconda-pre-shutdown.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
Shouldn't we use bash here too?

Reply:
That's a tough subject. Sorry I don't have a tl;dr.

I originally believed the hashbang mattered, but now I believe it does not, in dracut hooks at least. That's because dracut sources the hooks instead of calling. We do know that bashisms work, so perhaps we should go for bash everywhere, except dracut wants hooks to be POSIX compliant, so perhaps we should go for that instead?

However harmonizing the hashbangs could be an useless project.

Shellcheck changes rules depending on the hashbang, so it matters to some extent. Here I set it to the more strict mode, since the file did pass that as-is. Elsewhere I changed to bash instead, so as to make the file formally compliant and avoid more changes.

Reply:
I wouldn't really like ta change it everywhere or something similar. To me it's just strange that we have `bash` everywhere except this file in the commit.